app/views/users/index.html.erb

```diff
@@ -2,10 +2,11 @@
   <ul class="users-list">
     <% @users.each do |user| %>
       <li>
-        Name: <%= user.name %>
-        <span class="profile-link">
-          <%= link_to 'See Profile',  user_path(user), class: 'profile-link' %>
-        </span>
+        <%= user.name %></br>
+          <%= link_to 'See Profile',  user_path(user), class: 'btn btn-outline-success btn-sm' %>
+        <% if signed_in? && !Friendship.reacted?(current_user.id, user.id) && current_user != user %>
+          <td> <%= link_to 'Add Friend', friendships_create_path(ids: {id1: current_user.id, id2: user.id}), class: 'btn btn-success btn-sm' %></td>
+		<% end %>
```

Comment:
When you add a friend, you hide the button but you don't show the `pending_invitation` button. Please include it.